Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionPermission.h

```diff
@@ -75,6 +75,10 @@ WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionNativeMessagi
 WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA))
 WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionScripting;
 
+/*! @abstract The `sidePanel` permission requests access to the `chrome.sidePanel` APIs. */
+WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA))
+WK_EXTERN _WKWebExtensionPermission const _WKWebExtensionPermissionSidePanel;
```

Comment:
Move to `…Private.h`.

Reply:
Ah that was it!

Reply:
Unfortunately, 'Check For Inappropriate Macros in External Headers' is still failing if I use `ENABLE` in `_WKWebExtensionPermissionPrivate.h`. Should I just leave this symbol visible in all cases?

Reply:
i think that's fine